x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -368,6 +372,9 @@ export async function create(
     validateDuplicatedCustomFieldKeysInRequest({
       requestCustomFields: validatedConfigurationRequest.customFields,
     });
+    validateRequiredCustomFieldInRequest({
```

Comment:
Should we throw an error if a user tries to set a default value (even `null`) for an optional field?